test/integration/master/master_test.go

```diff
@@ -73,6 +73,16 @@ func TestExtensionsPrefix(t *testing.T) {
 	testPrefix(t, "/apis/extensions/")
 }
 
+func TestKubernetesService(t *testing.T) {
+	config := framework.NewMasterConfig()
+	_, _, closeFn := framework.RunAMaster(config)
```

Comment:
Yes, that's the plan. Haven't looked in detail over all the RunAMaster invocations. I guess most of them can just use the real master instead of the custom, non-standard bootstrapping.